### PR TITLE
image-components

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please Review my image components**
Images

### CODE REVIEW

File: Components/Images/images.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    box-shadow: 0px 0px 5px 2px rgba(0, 0, 0, 0.1);
+    flex-direction: column;
+   padding: 2rem;
+   flex-wrap: wrap;
+}
+.responsive-image{
+
+    width: 30rem;
+    height: 20rem;
+        box-shadow: 0px 0px 5px 2px rgba(0, 0, 0, 0.1);
```
Comment (zabihshaik) [inline Components/Images/images.css:21]: Better to use Prettier to format code

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. 
Just a suggestion to use Prettier to format Code to look good.

File: Components/Images/images.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,700;0,900;1,300&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="images.css">
+</head>
+<body>
+    <h2>Layouts Of Responsive Image and Round Image Components</h2>
+    <div class="images-container">
+        <div class="">
```
Comment (chanchal16) [inline Components/Images/images.html:17]: If you are not specifying any class better to remove it. The rest looks good.

Comment (anshtripathi01) [reply Components/Images/images.html:17]: thank you i remove it

**Pull request merged**